### PR TITLE
Todolist

### PR DESCRIPTION
**Pull request by imhilla**
Please review this project for JavaScript section of the curriculum.
We added all functionalities required and would wish for great feedback. 
We added:-
- Delete todo function
- Create a project and add task
- Added color functionality

### CODE REVIEW

File: src/task.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    this.title = title;
+    this.description = description;
+    this.priority = priority;
+    this.notes = notes;
+    this.duedate = duedate;
+    this.addTasks = () => [project, title, description, priority, notes, duedate];
+  }
+  return {
+    toDoItems,
+  };
```
Comment (trekab) [inline src/task.js:13]: Use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)) for this function declaration.

File: src/color.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable func-names */
+
+const colorModule = (() => {
+  const dropObj = ['black'];
+  function defaultColor() {
+    const styleColor = document.getElementById('colorChange');
+    styleColor.style.backgroundColor = `${dropObj[0]}`;
+    const styleContent = document.getElementById('colorName');
+    styleContent.textContent = `${dropObj[0]}`;
+  }
```
Comment (trekab) [inline src/color.js]: Use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)) for this function declaration.

File: src/project.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const project = (() => {
+  function projects(project, color) {
+    this.projectName = project;
+    this.projectColor = color;
+    this.addProject = () => [this.projectName, this.projectColor];
+  }
```
Comment (trekab) [inline src/project.js:6]: Use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622602/)) for this function declaration.

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @imhilla and @MFahadUmer! Well done! Your project looks good. There are a few changes to make before moving forward though. These are:
- [ ] Improve your local setup instructions in your `README.md` file. "open index from chrome" is not explicit enough for the user checking your project without prior development knowledge.
- [ ] Making any changes to the app e.g. adding a new project or todo item requires one to refresh the whole page in order to see the changes. Please fix this. The changes should be seen in real-time.
- [ ] __[Optional]__ The forms used to create a project or a todo should have validations, you should not be able to create an empty project/todo
- [ ] Check out the comments in the code below.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**